docker/edge/http/stop.sh

```diff
@@ -1,10 +1,7 @@
-sh $(pwd)/delete.sh
-if [ $? -eq 0 ]; then
-    docker stop metricbeat && docker rm $_
-    docker stop edgebroker && docker rm $_
-    docker stop edgeworker && docker rm $_
-else
-     echo failed in delete Application
-fi
+docker stop metricbeat && docker rm $_
```

Comment:
[Minor] Write proper message in front of this file -- why this file is created for.

Reply:
Comment Incorporated